roles/cleanup_openstack/tasks/main.yaml

```diff
@@ -0,0 +1,109 @@
+---
+- name: Include required vars
+  ansible.builtin.include_vars:
+    file: "{{ item }}"
+  loop:
+    - roles/kustomize_deploy/defaults/main.yml
+    - roles/deploy_bmh/defaults/main.yml
+
+- name: Load architecture automation file
+  register: _automation
+  ansible.builtin.slurp:
+    path: "{{ cifmw_architecture_automation_file }}"
+
+- name: Prepare automation data
+  vars:
+    _parsed: "{{ _automation.content | b64decode | from_yaml }}"
+  ansible.builtin.set_fact:
+    cifmw_deploy_architecture_steps: >-
+      {{ _parsed['vas'][cifmw_architecture_scenario] }}
+
+- name: Clean up testing resources
+  ansible.builtin.include_role:
+    name: test_operator
+    tasks_from: cleanup
+
+- name: Set baremetal hosts facts
+  vars:
+    _cifmw_deploy_bmh_bm_hosts: >-
+      {{
+        cifmw_baremetal_hosts | default({}) | dict2items |
+        rejectattr('key', 'in', ['crc', 'controller', 'ocp']) |
```

Comment:
scared here, if always we will reject proper hosts. Let's see

Reply:
This filtering mechanism is similar to the one used in the role creating the BMH do I assumed that if it works there it should be good enough for here.